doc/onboarding.md

```diff
@@ -0,0 +1,197 @@
+## pre-setup
+
+Ensure everyone is added to https://github.com/orgs/nodejs/teams/collaborators
+
+
+## onboarding to nodejs
+
+### intros
+
+
+### **thank you** for doing this
+
+  * going to cover four things:
+    * local setup
+    * some project goals & values
+    * issues, labels, and reviewing code
+    * merging code
+
+
+### setup:
+
+  * notifications setup
+    * use https://github.com/notifications or set up email
+    * watching the main repo will flood your inbox, so be prepared
+
+
+  * git:
+    * make sure you have whitespace=fix: `git config --global --add core.whitespace fix`
+    * usually PR from your own github fork
+    * **See "Updating Node.js from Upstream"**
+    * make new branches for all commits you make!
+
+
+  * `#node-dev` on `chat.freenode.net` is the best place to be to interact with the TSC / CTC / other collaborators
+
+
+### a little deeper about the project
+
+  * collaborators are effectively part owners
+    * the project effectively has the goals of it's contributors
+
+
+  * but, there are some higher-level goals and values
+    * not everything belongs in core (if it can be done reasonably in userland, let is stay in userland)
+    * empathy towards users matters (this is in part why we onboard people)
+    * generally: try to be nice to people, but we do have the Code of Conduct for a reason
+
+
+### managing the issue tracker
+
+  * you have (mostly) free reign – don't hesitate to close an issue if you are confident that it should be closed
+    * this will come more naturally over time
+    * IMPORTANT: be nice about closing issues, let people know why, and that issues and PRs can be reopened if necessary
+    * Still need to follow the Code of Conduct.
+
+
+  * labels:
+    * generally sort issues by a concept of "subsystem" so that we know what part(s) of the codebase it touches, though there are also other useful labels.
+     * **See "Labels"**
+    * ctc-agenda if a topic is controversial or isn't coming to a conclusion after an extended time.
+    * `semver-{minor,major}` labels for gauging versions numbers by commits:
```

Comment:
Maybe just get rid of `for gauging versions numbers by commits`. If people don't know what "semver" is, those words will just confuse them. If they do know what "semver" is, then they're not needed.